### PR TITLE
Initial commit HW34

### PR DESCRIPTION
**Pull request by jetyyya**
Added: Homework for lesson 34

### CODE REVIEW

File: HW34/.gitignore [hunk 1]
```diff
@@ -0,0 +1,39 @@
+target/
+idea/
+!.mvn/wrapper/maven-wrapper.jar
+!**/src/main/**/target/
+!**/src/test/**/target/
+
+### IntelliJ IDEA ###
```
Comment (Bibi4kovIgor) [inline HW34/.gitignore:7]: It is not needed to store all of configs in .gitignore file, that there are not used

File: HW34/.gitignore [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+.idea/modules.xml
+.idea/jarRepositories.xml
+.idea/compiler.xml
+.idea/libraries/
+*.iws
+*.iml
+*.ipr
+
+### Eclipse ###
```
Comment (Bibi4kovIgor) [inline HW34/.gitignore:16]: It is not needed to store all of configs in .gitignore file, that there are not used

File: HW34/.gitignore [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+.apt_generated
+.classpath
+.factorypath
+.project
+.settings
+.springBeans
+.sts4-cache
+
+### NetBeans ###
```
Comment (Bibi4kovIgor) [inline HW34/.gitignore:25]: It is not needed to store all of configs in .gitignore file, that there are not used

File: HW34/.gitignore [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+/nbbuild/
+/dist/
+/nbdist/
+/.nb-gradle/
+build/
+!**/src/main/**/build/
+!**/src/test/**/build/
+
+### VS Code ###
+.vscode/
```
Comment (Bibi4kovIgor) [inline HW34/.gitignore:36]: It is not needed to store all of configs in .gitignore file, that there are not used

**Review by Bibi4kovIgor — CHANGES_REQUESTED** [review]

File: HW34/.gitignore (hunk 1 above)
Comment (jetyyya) [reply HW34/.gitignore:7]: учту на майбутнє

**Review by Bibi4kovIgor — APPROVED** [review]

**Pull request merged**